packages/client/src/collection-manager/Configuration/ImportXlsx/setUpdateXlsxCollectionField.tsx

```
@@ -0,0 +1,42 @@
+    ctx.setVisible(false);
+  };
+
+  return {
+    async onClick() {
+      await form.submit();
+      const values = cloneDeep(form.values);
+      if (values.autoCreateReverseField) {
+      } else {
+        delete values.reverseField;
+      }
+      delete values.autoCreateReverseField;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)